### DISCUSSION TITLE
手機的電磁波會致癌嗎

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#1063] [General]
https://3c.ltn.com.tw/news/32941

我記得提到手機的電磁波造成人類的危害，老師們都會提出平常最常照的可見光做舉例，但為什麼還是有那麼多這類的問題被提出，論照的時間跟強度，可見光都更勝一籌吧!但也有可能是還有其他的影響因素，只是我不知道。

**Comment by Valentinaa0214** [comment]
是一個很貼近生活的議題，
雖然電磁波對致癌還沒有太多的證實，
但我們還是應該謹慎一些比較好，
像使用手機打電話時，也最好改用耳機或擴音文章中提到的：使用手機打電話時，最好改用耳機或擴音就是不錯的做法。

**Comment by Pietra1226** [comment]
現在睡覺都不敢把手機放枕頭旁邊，都放超級遠，結果常常鬧鐘沒聽到直接睡過頭...。

**Comment by ru0372572** [comment]
怕手機鬧鐘吵到室友還是會把手機放在頭旁邊，早上聽到的時候才比較好速速關掉（雖然因為沒爬起來很容易又睡回去），但這些電磁波訊號真的會有嚴重影響嗎?感覺很多人研究這個問題，也持續一段時間了，不過沒有一個確切的解答，或需要更長時間的研究?

**Comment by ghost** [comment]
每次看到這種報導都...無言以對，應該電磁波只是一個總稱，他包含了平常各種常用的波，像微波、可見光、X光等，所以探討電磁波的危害？？？是指什麼何種波長的？所以不同波長的波對人體的傷害不同，要探討手機對人體有無害，就必須知道除了可見光，他還會發出怎樣的波長的波，我想文章提到的研究應該也是真的某部分的波段的波所做的分析吧

**Comment by orianna901001** [comment]
我是抱著寧可信其有不可信其無的概念，所以我也是不把手機放在枕頭旁邊的，因為其實我的親戚也有聽說把手機放在枕頭邊好幾年，年紀輕輕就得腦癌，雖然不知道這兩者之間是不是有關連，但還是小心為妙。

**Comment by Jamesmoneya** [comment]
我是覺得手機的電磁波真的沒有那麼恐怖，如果真的那麼恐怖，那不甚麼電器你應該都要避免並且遠離，我覺得你如果因為要避免手機的電磁波而去影響你的生活習慣，那這樣的成本代價就高過手機發出的電磁波了

**Comment by hardtagottaname** [comment]
我記得曾經看過一個研究表明，電磁波的輻射比自然輻射低得多，因此，我認為手機電磁波應該不會致癌。

**Comment by KevinCheng42** [comment]
我覺得手機產生的電磁波能量遠小於可見光，因此，手機電磁波對人體的影響應該不大。我們反而要注意手機成癮和在陰暗空間使用手機造成視力衰減，甚至是因為過度暴露於藍光下影響睡眠品質！

**Comment by CCello** [comment]
從現有資料來看，目前尚未有充分的科學證據表明手機的電磁波會導致致癌。世界衛生組織（WHO）也表示，目前沒有證據顯示手機使用與癌症之間存在明確的關聯性。然而，電磁波確實會產生熱效應，長期使用手機對頭部的照射可能會引起頭痛、疲勞等不適症狀。因此，為了減少長時間使用手機對身體的潛在影響，建議使用免提裝置或藍牙耳機來接聽電話，並適當減少手機使用時間。

**Comment by Superlan8** [comment]
手機電磁波危害真的相較自然光或可見光來說較小，我認為比較需要注意的是手機的藍光，對我們的眼睛造成的傷害十分巨大，盡量避免躺著滑手機，或是睡前關燈滑手機，我認識很多長輩，眼睛都有青光眼或白內障，適度用眼才是我們需要擔心的。

**Comment by a3240281370** [comment]
目前的科學研究結果顯示，手機產生的無線電磁波不被認為是致癌的。大多數科學研究機構和衛生組織，包括世界衛生組織（WHO）和美國國家癌症研究所（NCI），都表示目前沒有充足的證據支持手機電磁波與癌症之間存在直接的因果關係。
雖然手機產生的電磁波可能對身體產生一些影響，但這些影響通常被認為是非致癌性的。例如，一些研究表明長時間使用手機可能與頭部的熱感覺、睡眠障礙和認知功能下降等問題有關。然而，這些研究結果仍然存在爭議，並需要進一步的研究來得出確定性的結論。

**Comment by jim1743** [comment]
我認為手機發射的電磁波對人體是幾乎無害的，研究都證明顯示電磁波幾乎沒有對人體產生傷害，那去相信電磁波是有害的理論不是很可笑，再者，現在日常生活中幾乎都會暴露在電磁波的影響下，我覺得要完全避免接觸到電磁波是很困難的事。

**Comment by EPHMR0524** [comment]
這個議題常常在長輩的社交圈之中流傳，即使我常常和他們說電磁波對人體基本無害，他們都不太相信。現在生活中到處都有人在傳訊息和看影片，不太可能是有一個地方是能完全沒有電磁波的，但我從小到大接收了那麼多電磁波現在也還是身體健康，我想大概是真的無害的，且醫學上現在也找不到電磁波致癌的證據。

**Comment by YOGA012** [comment]
可見光的照射時間和強度通常都比手機電磁波更高，因此我也覺得應該更關注可見光的影響。然而，提出手機電磁波的問題可能是因為人們對於新興科技的不確定性和擔憂，以及對於長期暴露於這些技術中可能帶來的潛在風險的關注。此外，科學研究中可能還存在一些尚未完全理解的影響因素，我覺得這也可能是為什麼這類問題經常被提出的原因之一。總的來說，這些問題需要進一步的研究和討論，以便更好地了解它們的影響和風險。

**Comment by YuanWang08** [comment]
生活中其實就已經接觸比手機還強的電磁波了，世界上無處沒有不是電磁波，手機的電磁波已經算是弱的了，若是真的還是擔心，可已將家中的電子設備在不用的時候全部拔掉插頭，或是可以搬到人煙稀少的地方像是山林間或沙漠，吃到電子設備的電磁波可以性比較低。

**Comment by alexender0104** [comment]
關於手機電磁波是否會致癌，科學界目前尚未有明確的結論。世界衛生組織（WHO）曾對這個問題進行過評估，將手機電磁場歸類為"可能對人類致癌"的一類，但同時指出目前的科學證據尚不足以確定這一風險。這意味著雖然有一些研究顯示手機電磁波可能與某些腦部腫瘤有關，但這種關聯性尚不確定。我覺得為了減少潛在的風險，一些建議包括使用藍牙耳機或免提裝置來減少手機與頭部的接觸，限制孩子使用手機的時間，選擇輻射值較低的手機等。

**Comment by tshsu04224** [comment]
我也會怕電子產品的電磁波對人類或是大腦造成影響，雖然因為做作業或寫程式，不可避免地一天中大部分的時間都會花費在這些電子產品上，但還是希望能減少影響就盡量減少，因此睡覺時我都會讓手機盡量遠離我，一方面減少電磁波的影響，一方面鬧鐘響的時候也能有效的起身去關掉它，就可以避免關掉鬧鐘又睡回去的狀況。

**Comment by sandyho0307** [comment]
我認為對手機電磁波的擔憂主要來自於對未知的恐懼、不確定的科學證據和媒體的影響。雖然可見光對我們的影響似乎更大，但由於我們對它的認識更充分，反而不會那麼擔心。

**Comment by lilishh** [comment]
我想大眾對於害怕手機電磁波對人體造成影響的部分，應是因為電磁波是無形的東西，無法感知，而且有些農場新聞會散布這些假資訊，我認為要等科學界真正來證明手機電磁波對於人類是否有不好的危害前，是可以以小心為妙的。

**Comment by 710388** [comment]
應該不會吧，會的話也沒什麼辦法，畢竟每天都離不開手機了。
